src/test/java/com/google/api/generator/engine/writer/JavaWriterVisitorTest.java

```diff
@@ -322,7 +322,7 @@ public void writeBlockCommentStatement_basic() {
     String content = "this is a test comment";
     BlockComment blockComment = BlockComment.builder().setComment(content).build();
     CommentStatement commentStatement = CommentStatement.withComment(blockComment);
-    String expected = "/** this is a test comment */\n";
+    String expected = "/*\n" + "* this is a test comment\n" + "*/\n";
     commentStatement.accept(writerVisitor);
     assertEquals(writerVisitor.write(), expected);
   }
```

Comment:
Could we please add a test for something with "bad formatting" like short lines and new lines in-between, so we can see how/whether that will be formatted?

Reply:
1. The formatter will not reformat the short lines (verified in local), below example will just keep it is.
```
/*
 * this is 
 * a test comment
 */
``` 
2. new lines in between  like `this is \n\n a test comment` will also reserve the new lines by formatter. (which is useful because we have extra new line in the apache license)
```
/*
 * this is
 *
 * a test comment
 */
```
3. long line is also kept as it is. unless it's surrounded by `/** */`. So when we construct the multi line block comment, the new line is used to be as splitter and we should be carefule about each line.

Reply:
Good, could we please add a test that demonstrates these cases?

Reply:
Yes, short lines/ newLineInbetween were added in the unit test, thanks!